var/spack/repos/builtin/packages/py-pygobject/package.py

```diff
@@ -31,18 +31,25 @@ class PyPygobject(AutotoolsPackage):
 
     homepage = "https://pypi.python.org/pypi/pygobject"
 
-    # FIXME: This URL is no longer available for download from PyPi
-    url      = "https://pypi.python.org/packages/6d/15/97c8b5ccca2be14cf59a2f79e15e3a82a1c3408a6b76b4107689a8b94846/pygobject-2.28.3.tar.bz2"
+    url      = "http://ftp.gnome.org/pub/GNOME/sources/pygobject/2.28/pygobject-2.28.6.tar.bz2"
 
+    version('2.28.6', 'a43d783228dd32899e6908352b8308f3')
     version('2.28.3', 'aa64900b274c4661a5c32e52922977f9')
 
     extends('python')
+
     depends_on("libffi")
     depends_on('glib')
     depends_on('py-py2cairo')
     depends_on('gobject-introspection')
 
-    patch('pygobject-2.28.6-introspection-1.patch')
+    patch('pygobject-2.28.6-introspection-1.patch', when='@2.28.3:2.28.6')
+
+    # patch from https://raw.githubusercontent.com/NixOS/nixpkgs/master/
+    #                   pkgs/development/python-modules/pygobject/
+    #                   pygobject-2.28.6-gio-types-2.32.patch
```

Comment:
Nope, any line containing `(file|https?)://` gets excluded from the flake8 line length checks.